lettre/templates/archive.html

```diff
@@ -0,0 +1,40 @@
+<!-- wp:template-part {"slug":"header-archive","tagName":"header"} /-->
+
+<!-- wp:group {"tagName":"main"} -->
+<main class="wp-block-group"><!-- wp:group {"layout":{"inherit":true}} -->
+<div class="wp-block-group">
+<!-- wp:search {"label":"Search","showLabel":false,"placeholder":"Search archive...","width":100,"widthUnit":"%","buttonText":"Search","buttonPosition":"button-inside","buttonUseIcon":true,"style":{"border":{"radius":"190px","width":"1px"}},"borderColor":"primary","backgroundColor":"primary","textColor":"secondary"} /-->
```

Comment:
Archive page inherits the width, but search page is full width.
Should they be split and add `query-title` in archive page?

Also, placeholder and button text are not translatable. Remove to fallback defaults.